src/main/scala/com.gu.gibbons/services/interpreters/BonoboInterpreter.scala

```diff
@@ -41,12 +41,16 @@ class BonoboInterpreter(config: Settings,
   def getDevelopers(users: Vector[User]) = {
     val uss = users.grouped(99).toVector
     for {
-      keys <- uss.foldMapM(
+      devKeys <- uss.foldMapM(
         us =>
-          getItems(keysTable, AndCondition('bonoboId -> us.map(u => UserId.unwrap(u.id)).toSet, 'tier -> "Developer"))
+          getItems(keysTable, 'bonoboId <= us.map(u => UserId.unwrap(u.id)).toSet and 'tier <= "Developer")
```

Comment:
I am just a bit confused about  the '->' and '<=' here.  What do they mean exactly? Are we not checking for equality against the "Developer" string?

Reply:
Thanks for pointing this out Juan, my understanding is that the -> operator in the AndCondition is effectively "is", this AndCondition is part of Scanamo which is a Scala library that allows us to deal with DynamoDB. I replaced it with a different query syntax but you are right that it is confusing and possibly incorrect so I pushed a fix where I kept the old syntax and just added a second query that is filtering for non-developers.